App/Containers/Styles/RenameGroup.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native'
+
+import Colors from '../../Themes/Colors'
+import { color } from '../../styles'
+
+export default StyleSheet.create({
```

Comment:
Let's just define all styles as `const`s within the file that uses them. And use explicit typescript types when defining them. There are plenty of example of this in the code base, but let me know if that doesn't make sense.